Rakefile

```diff
@@ -74,3 +72,107 @@ Rake::RDocTask.new do |rdoc|
   rdoc.rdoc_files.include('README*')
   rdoc.rdoc_files.include('lib/**/*.rb')
 end
+
+Rake::TestTask.new(:generate) do
+  $:.unshift File.join(File.dirname(__FILE__), "lib")
+  require "safemode"
+  default_methods = Safemode.class_variable_get("@@default_methods")
+  dangerous_methods = %w(
+  !~
+  =~
+  class
+  class_eval
+  clone
+  deep_clone
+  deep_dup
+  define_singleton_method
+  display
+  dup
+  enum_for
+  extend
+  gem
+  hash
+  __id__
+  inspect
+  instance_eval
+  instance_exec
+  instance_of?
+  instance_variable_defined?
+  instance_variable_get
+  instance_variables
+  instance_variable_set
+  itself
+  marshal_dump
+  marshal_load
+  method
+  methods
+  object_id
+  pathmap
+  pathmap_explode
+  pathmap_partial
+  pathmap_replace
+  pp
+  prepend
+  private_methods
+  protected_methods
+  public_method
+  public_methods
+  public_send
+  remove_instance_variable
+  __send__
+  send
+  silently
+  singleton_class
+  singleton_method
+  singleton_method_added
+  singleton_methods
+  taint
+  tainted?
+  tap
```

Comment:
this is probably safe and could be useful

Reply:
I think so, putting it back.